parallel_algorithms/P3179/P3179.md

```diff
@@ -1253,11 +1253,30 @@ that is sized and allows random access to its elements.
 ```
 :::
 
+## Modify [algorithms.requirements]{- .sref} ## {#modify_alg_reqs}
+
+[13]{.pnum} In the description of algorithm return values, a sentinel value `s` denoting the end of a range [`i`, `s`) is sometimes returned
+where an iterator is expected. In these cases, the semantics are as if the sentinel is converted into an iterator using
+`ranges​::​next(i, s)`.
+
+[14]{.pnum} Overloads of algorithms that take range arguments ([range.range]) behave as if they are implemented by[ calling
+ranges​::​begin and ranges​::​end on the range(s) and]{.rm} dispatching to the overload in namespace `ranges` that takes separate
+iterator and sentinel arguments[.]{.rm}[, where for each range argument `r` a corresponding:]{.add}
```

Comment:
Would not it be better (easier to read) if "a corresponding" is moved to each sub-bullet?

Reply:
Resolved by LWG review